### PR TITLE
Update source to openquake inputs

### PR DESCRIPTION
**Pull request by wkhchow**
Did a merge earlier on model-factory master to start working on the new changes in dsra / hexbin when I started noticing a bunch of files were missing (ie MH hazard prioritization scripts).  Then released there we did a PR/merge on opendrr-api branch but didn't follow through with model-factory.  I assume we need/should merge this to master as well?

### CODE REVIEW

File: scripts/Create_all_tables_update.sql
Comment (drotheram) [inline scripts/Create_all_tables_update.sql:128]: Should this value be added to the cesus source data?

**Review by drotheram — APPROVED** [review]
Looks good to me. Waiting on rebase to merge

**Review by anthonyfok — COMMENTED** [review]
I wonder if I could "Approve" this review after merging?  :-)

**Pull request merged**
